app/models/manageiq/providers/infra_manager/cluster.rb

```diff
@@ -0,0 +1,3 @@
+class ManageIQ::Providers::InfraManager::Cluster < ::EmsCluster
+  default_value_for :cloud, false
```

Comment:
Did you want to add a column for this in the schema PR?  As it is right now I get an error during refresh:
```
ERROR -- : [NoMethodError]: undefined method 'cloud=' for #ManageIQ::Providers::Vmware::InfraManager::Cluster:0x0000561b55f03d10>
```

Is there such a thing a "cloud" cluster?

Reply:
Copy/paste error from another implementation of subclassing.  Deleting...

Reply:
:+1: